md/message-states.md

```diff
@@ -14,3 +35,18 @@ All messages in the system are in one of the following states.
 | denied | This will not be sent as the recipient phone number is configured in the `outgoing_deny_list`. |
 | cleared | This will not be sent as a report has triggered an event to stop it. This can happen if a patient visit has occurred before the visit reminder is sent. |
 | muted | This will not be sent as the task has been deliberately stopped. Messages in this state can be unmuted by user action. |
+
+## Timeline of the protocol
+
+Event | webapp status | gateway status
+------|---------------|---------------
+Report comes in | scheduled | ---
+Due date to send the message passes | pending | ---
+Gateway pings webapp and gets the message | forwarded-to-gateway | pending
+Gateway confirms it got the message from webapp | received-by-gateway | pending
```

Comment:
`received-by-gateway`? yeah it does I'm pretty sure https://github.com/medic/medic-api/blob/master/controllers/sms-gateway.js#L11